spec/tags/optional/capi/integer_tags.txt

```diff
@@ -1,25 +1,7 @@
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with big endian output converts a negative number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with big endian output with overflow converts a positive number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with big endian output with overflow converts a negative number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with big endian output with overflow converts a negative number exactly -2**(numwords*wordsize*8)
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with little endian output converts a positive number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with little endian output converts a negative number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with little endian output with overflow converts a positive number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with little endian output with overflow converts a negative number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with little endian output with overflow converts a negative number exactly -2**(numwords*wordsize*8)
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with native endian output converts a positive number
-fails:CApiIntegerSpecs rb_integer_pack with two's complement flag with input greater than 64 bits with native endian output converts a negative number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack converts zero
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with big endian output converts a positive number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with big endian output converts a negative number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with little endian output converts a positive number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with little endian output converts a negative number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with native endian output converts a positive number
+fails(can't handle packing longs yet):CApiIntegerSpecs rb_integer_pack with two's complement flag with input less than 64 bits with native endian output converts a negative number
```

Comment:
I'll give that a go. We probably want to deal with the underlying problem at some point or FIXNUM_P will fail in real world use as well.